src/components/RiskItem.test.tsx

```
@@ -1,31 +1,43 @@
-    onSelect: vi.fn(),
-    onRevise: vi.fn(),
-    onApply: vi.fn(),
-    isLoading: false,
-  };
+/* ─── Mock the zustand store ────────────────────────────────── */
+const undoRevisionMock = vi.fn();
+const redoRevisionMock = vi.fn();
+
+vi.mock("../store/useMouStore", () => ({
+  /* every selector receives the slice we specify here */
+  useMouStore: (sel: any) =>
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)